src/game_inventory.cpp

```diff
@@ -540,21 +540,23 @@ class comestible_inventory_preset : public inventory_selector_preset
                 const item &it = *loc;
                 const nutrients nutr = p.compute_effective_nutrients( it );
                 const int kcalories = nutr.kcal;
-                //quit prematurely if there is no caloric content/the item is not food.
+                // Quit prematurely if there is no caloric content/the item is not food.
                 if( kcalories == 0 || !it.type->comestible ) {
                     return std::string();
                 }
                 units::volume water_vol = ( it.type->comestible->quench > 0 ) ? it.type->comestible->quench *
                                           5_ml : 0_ml;
-                units::volume food_vol = it.volume() - water_vol; //water volume is ignored.
+                // Water volume is ignored.
+                units::volume food_vol = it.volume() - water_vol;
                 const double converted_volume = round_up( convert_volume( food_vol.value() / it.count() ), 2 );
                 const double energy_density_ratio = p.compute_effective_food_volume_ratio( it );
                 const double effective_volume = converted_volume * energy_density_ratio;
                 const int calories_per_effective_volume = std::round( kcalories / effective_volume );
-                //arbitrary max value we will cap our vague display to. Will be lower than the actual max value, but it doesn't matter that much.
-                constexpr int max_cal_per_effective_vol =1500;
+                // Arbitrary max value we will cap our vague display to. Will be lower than the actual max value, but scaling fixes that.
+                constexpr int max_cal_per_effective_vol = 1500;
                 const int scaled_max = std::sqrt( max_cal_per_effective_vol ) / 4;
                 const int scaled_cal = std::sqrt( calories_per_effective_volume ) / 4;
+                // TODO: find a way to make this rtl aligned!
                 return get_hp_bar( std::min( scaled_max, scaled_cal ),
                                    scaled_max ).first;
```

Comment:
I _think_ this should do the rtl (also forcing increased granularity with "\\" and allows to change width):
```c++
std::string bar = get_bar( scaled_cal, scaled_max, 5, true).first; 
return std::string( bar.rbegin(), bar.rend() );
```

Reply:
![image](https://user-images.githubusercontent.com/25616971/94155134-459a4680-fe87-11ea-8984-6817e5734eec.png)

Your result: it reverses and aligns left-to-right.

Reply:
![image](https://user-images.githubusercontent.com/25616971/94155368-8c883c00-fe87-11ea-875a-e3abb9df816d.png)

My horrible hack: Aligns right-to-left without reversing, but still looks a bit ugly. Perhaps I should align to center?

Reply:
```c++
auto bar = get_bar( scaled_cal, scaled_max, 5, false); 
return colorize( std::string( 5-bar.first.length(), " " ) + bar.first, bar.second );
```

Reply:
Why do you say it has to be reversed? I think it makes more sense if it is not reversed.

Reply:
Your latest solution: 
![image](https://user-images.githubusercontent.com/25616971/94157048-6ebbd680-fe89-11ea-8f74-0b207ff10e93.png)

Looks pretty good, but still don't understand why it has to be reversed and aligned left-to-right. Also, for some reason, "\" doesn't get displayed any more.

Reply:
Not saying that.... Your "+++++" version did that initially. Ultimately this is going to come down to "I like _that_ one_ better".

This is you baby - pad with " " or ".", make it right, center or left aligned. Keep it with the "\" half indicator or not. And check if coloring is appropriate or not.

Reply:
Well, my "+++" version was admittedly horrible, something I threw together in a couple of minutes. I'll incorporate some of your ideas into my solution.

Reply:
Is there any way to add you as co-contributor to this PR?

Reply:
> for some reason, "\" doesn't get displayed any more.

4th parameter in `get_bar` - `true` does the half value, `false` omits them. 

Poor 🍿 - not even a _single_ bar :grin: